core/src/test/java/org/frankframework/http/rest/ApiListenerServletTest.java

```diff
@@ -382,6 +398,20 @@ public void apiListenerThatProducesJSON() throws ServletException, IOException,
 		assertNull(result.getErrorMessage());
 	}
 
+	@Test
+	public void apiListenerThatProducesJSONForMethodHead() throws ServletException, IOException, ListenerException, ConfigurationException {
+		String uri = "/ApiListenerThatProducesJSON/";
+		new ApiListenerBuilder(uri, Methods.HEAD, null, MediaTypes.JSON).build();
+
+		Response result = service(createRequest(uri, Methods.HEAD, "{}"));
```

Comment:
Does the 3rd parameter (content) should be null or an empty String?

Reply:
I do think an empty body, providing null gives both warnings/errors and looks a bit odd to me.

Reply:
Ah yes, this is input, instead of output.. it is fine 😉

Reply:
It's the request content, which is not even set when method = HEAD. Right?
So you should pass NULL I think.

I wonder why that would give errors?